assets/js/data/checkout/test/reducer.ts

```diff
@@ -26,7 +26,6 @@ describe.only( 'Checkout Store Reducer', () => {
 		const expectedState = {
 			...defaultState,
 			redirectUrl: 'https://example.com',
-			status: STATUS.IDLE,
```

Comment:
The reason for removing all the STATUS.IDLEs here is because we got rid of the reducer code that sets the status to IDLE for the first action performed on the store. That was a sideeffect essentially so we don't need to mock that in the test anymore